packages/mud-react/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@latticexyz/mud-react",
```

Comment:
wasn't sure what to call this! `@latticexyz/react` felt incorrect and I think what I actually want is something like `@mud/react`